Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN wget https://musl.cc/aarch64-linux-musl-cross.tgz \
     && tar zxf x86_64-linux-musl-cross.tgz \
     && rm -f *.tgz
 
-ENV PATH="/x86_64-linux-musl-cross/bin:/aarch64-linux-musl-cross/bin:/riscv64-linux-musl-cross/bin:$PATH"
\ No newline at end of file
+ENV PATH="/x86_64-linux-musl-cross/bin:/aarch64-linux-musl-cross/bin:/riscv64-linux-musl-cross/bin:$PATH"
```

Comment:
is this modification caused by encoding format?

Reply:
I enable `Insert Final Newline` in vscode and recommend every source file should end with `\n`.